views/groupDelete.njk

```diff
@@ -0,0 +1,58 @@
+{% extends "./partials/layout.njk" %}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/table/macro.njk" import govukTable -%}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set title =  "Delete group" %}
+
+ {% block beforeContent %}
+   {{ govukBreadcrumbs({
+     items: [
+       { text: "Home", href: homeUrl },
+       { text: "Manage user accounts", href: '/' },
+       { text: 'Manage groups', href: maintainUrl },
+       { text: groupDetails.groupName, href: groupUrl },
+       { text: title }
+     ]
+   }) }}
+ {% endblock %}
+{% block content %}
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ title }}</h1>
+
+      <p class="govuk-body govuk-!-margin-bottom-5">Do you want to delete the group - <STRONG>{{ groupDetails.groupName }}</STRONG></p>
+      <p class="govuk-body govuk-!-margin-bottom-5">All the user membership of this group will be removed</p>
+      <p class="govuk-body govuk-!-margin-bottom-9"><STRONG>This action cannot be undone</STRONG></p>
+      {% if hasMaintainAuthUsers %}
+
+        {{ govukButton({
+          text: "Delete group",
+          element: 'a',
+          href: maintainUrl + '/' + group + '/delete',
+          classes: "govuk-button--warning",
+          attributes: { "data-qa": "delete-group-button" }
```

Comment:
have a look at the other cancel button pages - I added some more spacing between the primary and secondary buttons

Reply:
will do thanks